combat-trainer.lic

```diff
@@ -4550,25 +4665,33 @@ class GameState
 
   # Determine the weapon skill to train offhand while aiming a ranged weapon.
   def determine_aiming_skill
+    damaris_weapon_options = get_damaris_weapon_options(weapon_name)
     options = @aiming_trainables
     options -= [weapon_skill] # can't be your mainhand skill, it's in your main hand
     options -= $twohanded_skills # can't be a twohanded skill, you need both hands
     options -= $aim_skills # can't use aimable weapons in your offhand
     options -= $melee_skills if weapon_skill.eql?('Bow') || (weapon_skill.eql?('Crossbow') && !@using_light_crossbow) # can't aim with weapon in your offhand
     options -= $thrown_skills if (weapon_skill.eql?('Bow') && @left_hand_free)
     options -= weapon_training.select { |skill, weapon| weapon == weapon_training[weapon_skill] }.keys # can't be same weapon that's equipped in your mainhand
+    options -= weapon_training.select { |skill, weapon| damaris_weapon_options.include?(weapon) }.keys # remove all weapons part of the same Damaris weapon group since they're the same item
+    options -= weapon_training.select { |skill, weapon| swappable_weapon?(weapon) }.keys # remove swappable weapons because they might be swapped into a two-handed state
```

Comment:
The swappable and summoned weapon checks would be good candidates for a separate PR. Not necessary, though.